ios/KMMICKIFUITests/Sources/Specs/Presentation/Module/SurveyDetailScreenSpec.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  SurveyDetailScreenSpec.swift
+//  KMMICKIFUITests
+//
+//  Created by MarkG on 16/05/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import Factory
+import Nimble
+import Quick
+
+@testable import KMMIC
+
+final class SurveyDetailScreenSpec: QuickSpec {
+
+    override func spec() {
+
+        describe("a SurveyDetail screen") {
+
+            var navigator: NavigatorMock!
+            var viewModel: SurveyDetailViewModelMock!
+            var didSetScreen = false
+
+            beforeEach {
+                guard !didSetScreen else { return }
+                didSetScreen = true
+                navigator = .init()
+                viewModel = .init(survey: .dummy)
+                viewModel.underlyingSurvey = .init(survey: .dummy)
+
+                self.app().setScreen(
+                    screen: .surveyDetail(viewModel: viewModel),
+                    navigator: navigator
+                )
+            }
+
+            it("has background image") {
```

Comment:
With a normal UITest target, for each `it` block, it will rerun a new session which takes time for UITests. How does it work with KIF? Please record the KIF tests and see if it rerun the app with multiple sessions

Reply:
@suho Only one instance of the app 🙏 

https://github.com/markgravity/kmm-ic/assets/17875522/a0ef3046-5418-4bc4-8244-62b89f2bc08b